code/game/objects/structures/mannequin.dm

```diff
@@ -0,0 +1,1386 @@
+		if(prob(40))//And 40% chance to get each robot limb back.
+			new part(loc)
+
+	if(prob(40))
+		var/obj/item/weapon/circuitboard/airlock/C = new(loc)
+		C.one_access=!(req_access && req_access.len>0)
+		if(!C.one_access)
+			C.conf_access=req_access
+		else
+			C.conf_access=req_one_access
+	..()
+
+/obj/structure/mannequin/cyber/ex_act(severity)
+	switch(severity)
+		if (1)
+			if(destroyed)
+				qdel(src)
+			else
+				destroyed = 1
+				getFromPool(/obj/item/weapon/shard, loc)
+				playsound(get_turf(src), "shatter", 100, 1)
+				shield = 0
+				update_icon()
+		if (2)
+			if (prob(50))
+				getDamage(30)
+			else
+				getDamage(20)
+		if (3)
+			if (prob(50))
+				getDamage(10)
+
+/obj/structure/mannequin/cyber/getDamage(var/damage)
+	if(destroyed || !locked)
+		health -= damage
+	else
+		shield -= damage
+	healthcheck()
+
+/obj/structure/mannequin/cyber/blob_act()
+	if(!destroyed && locked)
+		getDamage(30)
+	else if (prob(75))
+		qdel(src)
+	else
+		getDamage(30)
+
+/obj/structure/mannequin/cyber/healthcheck()
+	if(!destroyed)
+		if(health <= 100)
+			destroyed = 1
+			locked = 0
+			getFromPool(/obj/item/weapon/shard, loc)
+			playsound(get_turf(src), "shatter", 100, 1)
+			update_icon()
+		else
+			playsound(get_turf(src), 'sound/effects/Glasshit.ogg', 75, 1)
+	else
+		if(health <= 0)
+			visible_message("\The [src] collapses.")
+			qdel(src)
+
+/obj/structure/mannequin/cyber/attackby(var/obj/item/weapon/W,var/mob/user)
+	if(istype(W, /obj/item/weapon/card/id))
+		if(destroyed)
+			to_chat(user, "<span class='warning'>There is no longer any lock to toggle.</span>")
+		else
+			var/obj/item/weapon/card/id/I=W
+			if(!check_access(I))
+				to_chat(user, "<span class='rose'>Access denied.</span>")
+				return
+			locked = !locked
+			if(!locked)
+				to_chat(user, "[bicon(src)] <span class='notice'>\The [src] clicks as locks release, and it slowly opens for you.</span>")
+			else
+				to_chat(user, "[bicon(src)] <span class='notice'>You close \the [src] and swipe your card, locking it.</span>")
+			update_icon()
+	else if(iscrowbar(W) && (!locked || destroyed))
+		user.visible_message("[user.name] pries \the [src] apart.", \
+			"You pry \the [src] apart.", \
+			"You hear something pop.")
+		var/turf/T=get_turf(src)
+		playsound(T, 'sound/items/Crowbar.ogg', 50, 1)
+
+		if(do_after(user, src, 100))
+			var/obj/item/weapon/circuitboard/airlock/C = new(src)
+			C.one_access=!(req_access && req_access.len>0)
+
+			if(!C.one_access)
+				C.conf_access=req_access
+			else
+				C.conf_access=req_one_access
+
+			if(!destroyed)
+				getFromPool(/obj/item/stack/sheet/glass/glass, T, 1)
+
+			C.forceMove(T)
+
+			var/obj/structure/mannequin_frame/new_frame = new(T)
+			new_frame.icon_state = "mannequin_cyber_human"
+			new_frame.overlays |= image(icon, "lightout")
+			new_frame.construct = new /datum/construction/reversible/mannequin(new_frame)
+			qdel(src)
+
+	else if(istype(W, /obj/item/weapon/weldingtool) && (user.a_intent == I_HELP))
```

Comment:
> Here in this case you could say that "&&" is a "+" and "==" is a "x"

Sorry but what? Now you're just making crap up for the sake of not having to change it.

> Logical expression priorities work the same way that arithmetic ones do.

... Only if you come up with dumb logic. It can be argued about the `+`/`&&` but `==`? Not really.